specifications/network/network-address.md

```diff
@@ -49,25 +46,20 @@ pub enum Protocol {
 /// protocol delimiter and Rust's [`::std::net::ToSocketAddr`] API requires a
 /// `&str`.
 pub struct DnsName(String);
-
-/// A `RawNetworkAddress` is the serialized, unverified, on-chain representation
-/// of a [`NetworkAddress`]. Specifically, a `RawNetworkAddress` is an
-/// [`lcs`]-serialized `NetworkAddress`.
-///
-/// Note: deserializing a `RawNetworkAddress` does no validation, other than
-/// deserializing the underlying `Vec<u8>`.
-pub struct RawNetworkAddress(Vec<u8>);
 ```
 
 (TODO(philiphayes): link to x25519 ser/de spec)
-(TODO(philiphayes): hardening to limit NetworkAddress and RawNetworkAddress max size)
+(TODO(philiphayes): hardening to limit NetworkAddress max size)
 
 ## Human-readable Format
 
-All `NetworkAddress` sent and received over-the-wire or stored on-chain are in their compact `RawNetworkAddress` [`lcs`]-serialized format. However, implementations may also wish to implement the optional human-readable format to aid reading `NetworkAddress`es from configuration or printing `NetworkAddress` in logs.
+All `NetworkAddress` sent and received over-the-wire or stored on-chain are in their compact [`u8`] [`lcs`]-serialized format. However, implementations may also wish to implement the optional human-readable format to aid reading `NetworkAddress`es from configuration or printing `NetworkAddress` in logs.
 
-Each `Protocol` segment is formatted like:
+Each `Protocol` segment is formatted of the form `/<Protocol>/<Value>`, and applies to the following rules:
```

Comment:
personally, I view these as `/<Protocol>[/<Argument>]*`, where you can have zero-or-more "arguments" for each protocol. for example, `/memory/<port>` takes 1 argument. of course all of our protocols happen to only take one argument atm : p

Reply:
Well, I didn't say anywhere that `<Value>` had to be a single item, it could be encoded however we like, but there's currently no specification on how you would separate multiple arguments because there's no example.

I have a feeling that's up to the protocol to specify?

Reply:
I've gone ahead and changed the spec to say "ProtocolConfig" instead of "Value"